plugins/rollbar/app/models/rollbar_notification.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+require 'faraday'
+
+class RollbarNotification
+  def initialize(webhook_url:, access_token:, environment:, revision:)
+    @webhook_url = webhook_url
+    @access_token = access_token
+    @environment = environment
+    @revision = revision
+  end
+
+  def self.deliver(webhook_url:, access_token:, environment:, revision:)
+    new(
+      webhook_url: webhook_url,
+      access_token: access_token,
+      environment: environment,
+      revision: revision
+    ).deliver
+  end
+
+  def deliver
+    Rails.logger.info "Sending Rollbar notification..."
+    response = Faraday.post(@webhook_url,
```

Comment:
I've updated it to match the suggested style, Rubocop should be updated to enforce this :-)